Added 12 new Q&A pairs to the Support knowledgebase section (#1081)
wandb/docs: content/support/what_will_happen_if_i_edit_my_python_files_while_a_sweep_is_running.md

---

> title: What will happen if I edit my Python files while a sweep is running?

Reviewer: ```suggestion
title: What happens if I edit my Python files while a sweep is running?
```

toc_hide: true
type: docs
tags:
  - sweeps
---

> There are two things could change:
> - The `train.py` which the sweep uses
> - The files that `train.py` references (such as other scripts, let’s say a `helper.py` with helper functions is imported in `train.py`)
> 
> If you change `train.py` during a sweep, the sweep will continue to use the original version of `train.py`. It will not use the updated version of `train.py`.
> 
> However, if you change `helper.py` during the sweep, the sweep will use the newer version of `helper.py`.

Reviewer: ```suggestion
While a sweep is running:
- If the `train.py` script which the sweep uses changes, the sweep continues to use the original `train.py`
- If files that the `train.py` script references change, such as helper functions in the `helper.py` script, the sweep begins to use the updated `helper.py`.
```
